Extension/package.json

```diff
@@ -308,6 +308,19 @@
           "description": "Determines whether pop up notifications will be shown when a configuration provider extension is unable to provide a configuration for a source file.",
           "scope": "resource"
         },
+        "C_Cpp.intelliSense.cachePath": {
```

Comment:
This naming style seems inconsistent with our existing settings like intelliSenseEngine, intelliSenseMode, and intelliSenseEngineFallback.

Reply:
Yeah, we were trying to be forward-looking, but I agree this might be premature.
We should rename these `intelliSenseCachePath` and `intelliSenseCacheSize`.